src/scribe_data/utils.py

```diff
@@ -240,6 +244,66 @@ def get_language_words_to_ignore(language: str) -> list[str]:
     )
 
 
+def get_language_dir_path(language):
+    """
+    Constructs the directory path for a given language's data within the project.
+
+    Parameters:
+    - language (str): The name of the language for which the directory path is needed.
+
+    Returns:
+    - str: The directory path for the specified language's data.
+    """
+    PATH_TO_SCRIBE_ORG = os.path.dirname(sys.path[0]).split("Scribe-Data")[0]
+    return f"{PATH_TO_SCRIBE_ORG}/Scribe-Data/src/scribe_data/extract_transform/languages/{language}"
+
+
+def load_queried_data(language, data_type, file_path):
+    """
+    Loads queried data from a JSON file for a specific language and data type.
```

Comment:
And for all three of the functions that you're adding, could you style the docstrings so that they resemble the other functions in the codebase? This will allow us to autodoc them more easily as we need them to by Numpy formatted docstrings for them to be formatted for the docs appropriately :)

Reply:
done.